velox/connectors/hive/PartitionIdGenerator.cpp

```diff
@@ -119,7 +119,7 @@ void PartitionIdGenerator::computeValueIds(
     }
   }
 
-  if (!rehash) {
+  if (!rehash && multiplierInitialized_) {
```

Comment:
```
// NOTE: for boolean column type, computeValueIds() always returns true and this might cause problem in case of multiple boolean partition columns as we might not set the multiplier properly.
```
Do we run into this issue if boolean value is null?
Also there is a bug in 
```
template <>
bool VectorHasher::makeValueIdsFlatWithNulls<bool>(
    const SelectivityVector& rows,
    uint64_t* result) {
  const auto* values = decoded_.data<uint64_t>();
  const auto* nulls = decoded_.nulls(&rows);
  rows.applyToSelected([&](vector_size_t row) INLINE_LAMBDA {
    if (bits::isBitNull(nulls, row)) {
      if (multiplier_ == 1) {
        result[row] = 0;
      }
      return;
    }
    bool value = bits::isBitSet(values, row);
    uint64_t id = valueId(value);
    result[row] = multiplier_ == 1 ? id : result[row] + multiplier_ * id;
  });
  return true;
}
```
What we expect in the return value if it is null? Thanks!